media/css/firefox/private_browsing/private-browsing-conditionals.less

```diff
@@ -0,0 +1,170 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+@import "../../sandstone/lib.less";
+
+.download-buttons  {
+    > div {
+        display: none;
+    }
+
+}
+/*-------------------------------------------------------------------------*/
+// Android Links and Download hide/show
+
+.android {
+    .footer .android-link {
+        display: block;
+    }
+    .footer .desktop-link {
+        display: none;
+    }
+    .download-buttons {
+    }
+    &.non-firefox .download-buttons .non-firefox-cta {
+        .desktop-download-button {
+            display: none;
+        }
+        h2 {
+            width: 100%;
+        }
+        .android-download-button {
+            display: block;
+        }
+    }
+}
+
+/*-------------------------------------------------------------------------*/
+// iOS Links and Download hide/show
+
+.ios {
+    .footer .ios-link {
+        display: block;
+    }
+    .footer .desktop-link {
+        display: none;
+    }
+
+    &.non-firefox .download-buttons .non-firefox-cta {
+        & .desktop-download-button {
+            display: none;
+        }
+        .ios-download-button {
+            display: block;
+        }
+    }
+}
+
+    
+/*-------------------------------------------------------------------------*/
+// Firefox Up to date download buttons show/hide
+
+.firefox-up-to-date {
+    .download-buttons .firefox-latest-cta {
+        display: block;
+    } 
+}
+
+
+/*-------------------------------------------------------------------------*/
+// Firefox Old download buttons show/hide plus styling
+
+.firefox-old {
+    .download-buttons .firefox-old-cta {
+       display: block; 
+    }
+    .cta {
+        .container {
+            padding: 60px 20px;
+        }
+        h2 {
+            .font-size(30px);
+            line-height: 1.4;
+            width: 440px;
```

Comment:
I do not believe you need to make the width smaller here. It can just use the `500px` from `.cta h2`. Making it smaller causes a disconnect between the message and the button I feel.

Currently:

![screen shot 2015-10-27 at 12 52 25](https://cloud.githubusercontent.com/assets/10350960/10755973/a9e11598-7ca9-11e5-8c4d-19fc62a9f4fa.png)

As apposed to:

![screen shot 2015-10-27 at 12 52 38](https://cloud.githubusercontent.com/assets/10350960/10755980/b286c0da-7ca9-11e5-91e1-3c0be806305e.png)